katalog/harbor/jobservice/kustomization.yaml

```
@@ -9,27 +9,33 @@ kind: Kustomization
 resources:
   - deploy.yml
   - svc.yml
+  - pvc-logs.yml
+  - pvc-scandata.yml
 
 images:
   - name: goharbor/harbor-jobservice
     newName: registry.sighup.io/fury/goharbor/harbor-jobservice
-    newTag: v2.4.2
+    newTag: v2.7.0
 
 configMapGenerator:
   - name: jobservice
     files:
       - config.yml=config/config.yml
     literals:
       - CORE_URL=http://core
+      - REGISTRY_URL=http://registry:5000
+      - TOKEN_SERVICE_URL=http://core/service/token
       - REGISTRY_CONTROLLER_URL=http://registry:8080
       - LOG_LEVEL=info
+      - METRIC_NAMESPACE=harbor
```

shouldn't this be `registry` or is this another kind of namespace?
```suggestion
      - METRIC_NAMESPACE=registry
```

---

Hi @ralgozino , this parameter has not been set to "registry" as with this value the metrics are not displayed correctly and therefore visible on the grafana dashboard.